deb/build/debian/jenkins.service

```diff
@@ -0,0 +1,24 @@
+[Unit]
+Description=Jenkins Continuous Integration Server
+After=network.target
+
+[Service]
+Type=simple
+User=jenkins
+Group=jenkins
+SyslogIdentifier=jenkins
+UMask='0002'
```

Comment:
you can have secrets in places other than secrets... e.g. a `Secret` in a jobs `config.xml` where access to that can allow you to use it elsewhere and recover the secret for a job that you should have no access to.